tests/poison_detection/test_activation_defence.py

```diff
@@ -198,7 +198,7 @@ def test_pickle(self):
         ActivationDefence._pickle_classifier(self.classifier, filename)
         loaded = ActivationDefence._unpickle_classifier(filename)
 
-        self.assertEqual(self.classifier._clip_values, loaded._clip_values)
+        self.assertEqual(self.classifier._clip_values.tolist(), loaded._clip_values.tolist())
```

Comment:
Would it be possible to use `numpy.testing.assert_equal` for testing equality of numpy arrays